tasks/server_keys.yml

```diff
@@ -137,3 +137,19 @@
   args:
     chdir: "{{ openvpn_key_dir }}"
     creates: "{{ openvpn_key_dir }}/ca-crl.pem"
+
+- name: install crl-cron script
+  template:
+    src: crl-cron.sh.j2
+    dest: "{{ openvpn_base_dir }}/crl-cron.sh"
+    owner: root
+    group: root
+    mode: 744
+
+- name: Add cron to check every Saturday if the CRL needs to be renewed
+  cron:
```

Comment:
This is breaking the CI builds, so there needs to be an additional `when: not ci_build`
https://github.com/kyl191/ansible-role-openvpn/blob/master/defaults/main.yml#L24